lib/util/index.js

```diff
@@ -520,6 +520,40 @@ function valueToJSON(value) {
   return value;
 }
 
+/**
+ * Create the bandwidth profile payload included in an RSP connect message.
+ * @param {BandwidthProfileOptions} bandwidthProfile
+ * @returns {object}
+ */
+function createBandwidthProfilePayload(bandwidthProfile) {
+  const payload = {};
+  if (typeof bandwidthProfile.video !== 'object') {
+    return payload;
+  }
+
+  const mode = bandwidthProfile.video.mode;
+  if (typeof mode === 'string') {
+    payload.mode = mode;
+  }
+
+  const maxSubscriptionBitrate = bandwidthProfile.video.maxSubscriptionBitrate;
+  if (typeof maxSubscriptionBitrate === 'number') {
+    payload.video = Object.assign({
+      // eslint-disable-next-line
+      max_subscription_bandwidth: Math.round(maxSubscriptionBitrate / 1024)
+    }, payload.video);
+  }
+
+  const maxTracks = bandwidthProfile.video.maxTracks;
+  if (typeof maxTracks === 'number') {
```

Comment:
same thing - not sure what `Object.assign` is needed.

Reply:
@makarandp0 See my comment above.